.circleci/config.yml

```diff
@@ -36,8 +36,9 @@ jobs:
       - run:
           name: Install Python dependencies
           command: |
-            pip --version
+            pip --version 
             pip install --progress-bar off -r requirements/doc-requirements.txt pytest pytest-cov plotly .[gateway]
+            pip install tqdm ipywidgets
```

Comment:
```suggestion
            pip install --progress-bar off -r requirements/doc-requirements.txt pytest pytest-cov plotly .[gateway] tqdm ipywidgets
```

Reply:
Do we need `tqdm ipywidgets` when building docs?

Reply:
Ah now we don't :)